plugins/grpc/swift/v1.22.0/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# syntax=docker/dockerfile:1.4
```

Comment:
do we usually bump these when we can, or just when we need a new syntax feature?

Reply:
_when we can_, we've chatted about adding a Dockerfile and then reading the value and making this automated. But the ROI is kind of low tbh.

Reply:
Updated.